packages/commonwealth/client/scripts/navigation/GeneralRoutes.tsx

```diff
@@ -36,6 +37,11 @@ const GeneralRoutes = () => [
     path="/components"
     element={withLayout(ComponentsShowcasePage, { type: 'common' })}
   />,
+  <Route
```

Comment:
i think we can make a new DEV_MODE flag (or check NODE_ENV!='production') or non-user facing pages

Reply:
actually I will get rid of it as it was only for testing purpose